### PR TITLE
Thread view refresh

### PR DESCRIPTION
**Pull request by nschloe**
Fixes bug #387 and #267.

Also get live updates in discussion list view to work.

### CODE REVIEW

File: app/components/discussion-thread-view/index.ts [hunk 1]
```diff
@@ -7,51 +7,53 @@ export default function(app) {
...
-                }
-              ]
-            });
-          })
-          .error(notificationService.httpError('could not fetch discussion'));
+          const ctrl = this;
+
+          $scope.$watch('$ctrl.discussions.stored', function(discussions) {
+            // discussion with ID $routeSegment.$routeParams.discussionId
+            console.log(discussions);
```
Comment (andrenarchy) [inline app/components/discussion-thread-view/index.ts]: should be removed

**Comment by andrenarchy** [comment]
Replies that are added in the thread view appear twice.

**Comment by nschloe** [comment]
@andrenarchy All issues fixed now. Phew!

**Comment by andrenarchy** [comment]
I just found one more issue: updating a discussion or a reply in the pdf view does not update the thread view (in another browser window). 

Removing/adding replies is synced properly. Also syncing from the thread view to the pdf view works nicely. :+1:

**Comment by nschloe** [comment]
> I just found one more issue: updating a discussion or a reply in the pdf view does not update the thread view (in another browser window).

I've looked at it and didn't find the cause. Considering this is also in master, I'd vote for filing a bug and handling it separately.

**Comment by andrenarchy** [comment]
+1

**Comment by nschloe** [comment]
:+1:

**Pull request merged**